pyro/contrib/autoguide/__init__.py

```diff
@@ -85,6 +85,12 @@ def __call__(self, *args, **kwargs):
         """
         raise NotImplementedError
 
+    def get_posterior(self, *args, **kwargs):
```

Comment:
I believe `AutoGuide` should not have this method. The method appears specific to `AutoContinuous`. And certainly the method should not return None.

Reply:
Yup, agree. :)